alumnos/ocf/clase-05/11.js

```diff
@@ -0,0 +1,73 @@
+// 11. En casa tenemos una estantería muy interesante de libros. Cada libro se compone de las siguientes propiedades... 
+
+const estanteria = {
+  libros: [{
+    nombre: 'El caballero oscuro',
+    autor: 'Frank Miller',
+    leido: false
+  },
+  {
+    nombre: 'El mundo amarillo',
+    autor: 'Albert Espinosa',
+    leido: false,
+  },
+  {
+    nombre: 'Harry Potter y el caliz de fuego',
+    autor: 'J.K. Rowling',
+    leido: true,
+  },
+  {
+    nombre: 'El ingenioso hidalgo Don Quijote de la Mancha',
+    autor: 'Miguel de Cervantes',
+    leido: false
+  },
+  {
+    nombre: 'Berserk',
+    autor: 'Kentaro Miura',
+    leido: true
+  },
+  {
+    nombre: 'Iliada',
+    autor: 'Homero',
+    leido: false
+  }],
+  
+  infoLibro() {
+    let libros = [];
+
+    for (const libro of this.libros) {
+      let {nombre, autor, leido} = libro;
+      libros.push(libro);
+      let frase = ` has leido ${nombre} de ${autor}`;
+      console.log(leido ? `Ya ${frase}` : `Aún no ${frase}`);
+    }
+  },
+  
+  sugerencia() {
+    let noLeidos = [];
+    
+    for (const libro of estanteria.libros) {
+      const { nombre, autor, leido} = libro;
+      !leido ? noLeidos.push(libro) : false;
```

Comment:
Aquí con un if sería más legible y práctico

```suggestion
if (!leido) {
noLeidos.push(libro)
}
```

Reply:
Hecho!